README.md

```diff
@@ -142,7 +148,7 @@ This is a type for representing string values.
 The `get` method returns the value of the `StringType` instance.
 
 ```javascript
-var stringType = new minim.StringType('foobar');
+var stringType = new minim.StringType({}, {}, 'foobar');
```

Comment:
I've updated this in the latest commit. The order is now `ElementType(element, content, meta, attributes)` and the other types don't require an element, e.g. `StringType('some string');` just like before. The important note is that even the base element type has `content` as the second argument for consistency sake.